Formula/docutils.rb

```diff
@@ -4,6 +4,7 @@ class Docutils < Formula
   url "https://downloads.sourceforge.net/project/docutils/docutils/0.19/docutils-0.19.tar.gz"
   sha256 "33995a6753c30b7f577febfc2c50411fec6aac7f7ffeb7c4cfe5991072dcf9e6"
   license all_of: [:public_domain, "BSD-2-Clause", "GPL-3.0-or-later", "Python-2.0"]
+  revision 1
```

Comment:
We probably should have revision bumped back when we added `python@3.11` support.

Since we only rebuilt bottles, anyone who previously installed `docutils` with only `python@3.10` support will have broken installs for dependents that were migrated to `python@3.11`, e.g. `awscli`, `snakemake`, `rst-lint`, ...

Reply:
Maybe we should only depend on a formula once it builds for all current interpreters (like `six` etc)